app/src/main/kotlin/pl/gov/mc/protego/ui/base/BaseActivity.kt

```diff
@@ -13,4 +16,20 @@ abstract class BaseActivity : BaseVariantActivity() {
             }
         }
     }
+
+    fun showNoInternetConnectionDialog() {
+        supportFragmentManager.findFragmentByTag(NO_INTERNET_DIALOG) ?: NoInternetConnectionDialog().apply {
```

Comment:
powinno być:
```suggestion
        (supportFragmentManager.findFragmentByTag(NO_INTERNET_DIALOG) ?: NoInternetConnectionDialog()).apply {
```
tak jak test teraz to `apply{}` zadziała tylko przy tworzeniu obiektu i `findFragmentByTag()` jest zbędny bo jak istnieje to się nie pokaże